azure/services/resourceskus/sku.go

```diff
@@ -128,3 +130,23 @@ func (s SKU) GetCapability(name string) (string, bool) {
 	}
 	return "", false
 }
+
+// HasLocationCapabilities returns true if the provided resource supports the location capability.
+func (s SKU) HasLocationCapabilities(name, size, location string) bool {
```

Comment:
Maybe this should be `HasLocationCapability()` for consistency with the related functions? It's also more descriptive since it only checks for one capability.

Reply:
sounds fair